universal-application-tool-0.0.1/app/auth/oidc/AdfsProfileAdapter.java

```diff
@@ -64,7 +64,12 @@ protected void adaptForRole(CiviFormProfile profile, ImmutableSet<Roles> roles)
   }
 
   private boolean isGlobalAdmin(OidcProfile profile) {
-    JSONArray groups = profile.getAttribute("group", JSONArray.class);
+    JSONArray groups = (JSONArray) null;
+    if (profile.containsAttribute("group")) {
+      groups = profile.getAttribute("group", JSONArray.class);
+    } else if (profile.containsAttribute("groups")) {
+      groups = profile.getAttribute("groups", JSONArray.class);
+    }
```

Comment:
suggestion (blocking): this is fairly ambiguous code as to why it's behaving this way. Please add an application config var instead.

Reply:
cool yes good points.